Adding label selector for triggers (#970)
tektoncd/triggers: docs/eventlisteners.md

    - *
```

### LabelSelector
This field determines the labels for which EventListener will select triggers and
create Tekton resources.

> Snippet below will function the eventlistener namespace and only select triggers with 

Reviewer: ```suggestion
In the snippet below, the EventListener will only select triggers from its's  namespace with 
```

Reviewer 2: "its namespace" - this is a possessive, so no apostrophe ("it's" is short for "it is").

Author: yeah, updated to `its namespace`

> Snippet below will function the eventlistener namespace and only select triggers with 

Reviewer 2: Capitalize consistently - "EventListener"
